DiscordBot.Modules/CommandModules/CodeSnippetModule.cs

```diff
@@ -0,0 +1,261 @@
+﻿using Discord.Commands;
+
+using DiscordBot.Domain.Configuration;
+
+using Microsoft.Extensions.Options;
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+namespace DiscordBot.Modules.CommandModules
+{
+    public class SnippetInfo
```

Comment:
Separate class pls

Reply:
What do you mean? This is a seperate class! Or should I put it into a seperate file?

Reply:
Yea i meant file 😅

Reply:
Where should I put it? In DiscordBot.Domain/Abstractions ? And what should I add to the Program.cs ?